src/elements/content-sidebar/additional-tabs/AdditionalTabs.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @flow
+ * @file Preview sidebar additional tabs component
+ * @author Box
+ */
+
+import React, { PureComponent } from 'react';
+import AdditionalTab from './AdditionalTab';
+import AdditionalTabsLoading from './AdditionalTabsLoading';
+
+import './AdditionalTabs.scss';
+
+type Props = {
+    tabs: ?Array<AdditionalSidebarTab>,
```

Comment:
It looks like the parent component's flow type has this prop marked as optional rather than nullable.